docs/plugins.md

```diff
@@ -8,7 +8,7 @@ redirect_from:
  - /docs/usage/modules/
 ---
 
-Out of the box Babel doesn't do anything. In order to actually do anything to your code
+Out of the box Babel doesn't do anything. In order to actually do something to your code
```

Comment:
Maybe we should just remove that and just do something like `Out of the box Babel doesn't do anything; you need to enable plugins to transform your code.`